lib/foods.js

```diff
@@ -47,9 +52,25 @@ function tableFilter() {
 $('document').ready(function() {
   $("#add-food").on('click', function() {
     checkInputs();
-    loadButton();
+    createDeleteButton();
+    $('td').on('click', function() {
+      var $this = $(this);
+      var $input = $('<input>', {
+        value: $this.text(),
+        type: 'text',
+        blur: function() {
+          $this.text(this.value);
+        },
+        keyup: function(e) {
+          if (e.which === 13) $input.blur();
+        }
+      }).appendTo( $this.empty() ).focus();
```

Comment:
@lucyconklin Shouldn't this be 1}).appendTo( $this ).focus();` without `$this.empty()`? Wouldn't this empty `td`?